Shoko.Server/API/v3/Models/Shoko/Series.cs

```diff
@@ -593,4 +636,97 @@ public SeriesSearchResult(HttpContext ctx, SVR_AnimeSeries ser, string match, do
             Distance = dist;
         }
     }
+
+    public interface ISeriesMetadata {
```

Comment:
Again with the multiple interfaces/classes per file.

Reply:
Removed the interfaces.